ansible/roles/dashd/tasks/main.yml

```diff
@@ -123,14 +123,14 @@
 # Wallet node
 - name: Create faucet wallet
   ansible.builtin.command:
-    cmd: dash-cli createwallet {{ wallet_rpc_wallet_faucet }}
+    cmd: dash-cli createwallet {{ wallet_rpc_wallet_faucet }} false false "" false true #Reload wallet on restart
```

Comment:
Why does the wallet need to be reloaded through a `createwallet` command on startup? How is this different to the block on line 101?

Reply:
This ensures the wallet will load when dashd is restarted, if you restart dashd on wallet1 without this - it will not load the faucet wallet. 

I can put this in a separate PR since it's technically a bug fix

Reply:
Either the block on line 101 was added later (and ported in) or doesn't work for some reason. I will test and remove if it's not needed anymore

Reply:
@ktechmidas you added the 101 block yourself in https://github.com/dashpay/dash-network-deploy/pull/438

Did you test this? I haven't seen any evidence it doesn't work, and the RPC command will just edit `dash.conf` for us, so I'm going to remove this and rely on Ansible directly for templating instead of templating by proxy through dashd.